exercises/practice/connect/.meta/example.gleam

```
@@ -0,0 +1,152 @@
+import gleam/list
+import gleam/map.{Map}
+import gleam/string
+import gleam/result
+
+pub type Player {
+  X
+  O
+}
+
+type Position {
+  Position(row: Int, column: Int)
+}
+
+type Size {
+  Size(rows: Int, columns: Int)
+}
+
+pub fn winner(board: String) -> Result(Player, Nil) {
+  let rows =
+    board
+    |> string.trim()
+    |> string.split("\n")
+
+  assert [first_row, ..] = rows
```

I would return an error here, the board could be empty.